public/js/util/validators.js

```diff
@@ -1,44 +1,51 @@
 import FieldError from '../constants/FieldError';
+import { wordLimits } from './wordLimits';
+
 /**
  *
  * Validator should return a promise resolved with true for a pass and a new FieldError('error', 'message') if false
  *
  **/
 
-
-export const isHttpsUrl = (value) => {
-  const stringValue = typeof value === 'string' ? value : '';
-  if (stringValue.match(/^(https:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$/)) {
-    return Promise.resolve(true);
-  } else {
-    const error = new FieldError('not-https', 'Not a HTTPS url');
-    return Promise.resolve(error);
-  }
+export const isHttpsUrl = value => {
+	const stringValue = typeof value === 'string' ? value : '';
+	if (
+		stringValue.match(
+			/^(https:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$/
+		)
+	) {
+		return Promise.resolve(true);
+	} else {
+		const error = new FieldError('not-https', 'Not a HTTPS url');
+		return Promise.resolve(error);
+	}
 };
 
-const WordLimit = 400;
-export const checkItemsUnderWordCount = (values) => {
-    if (values.length > 0) {
-        var wordCount = values.map(function(itemData) {
-            var strippedText = stripHtml(itemData.body);
-            return strippedText.split(" ").length;
-        });
-
-        var totalCount = wordCount.reduce(function (total, num) {
-            return total + num;
-        });
-
-        if (totalCount <= WordLimit) {
-            return Promise.resolve(true);
-        } else {
-            const error = new FieldError('too-long', `You\'ve reached the word limit on this atom type. Please edit your items to less than a total of ${WordLimit} words.`);
-            return Promise.resolve(error);
-        }
-
-    }
+export const checkItemsUnderWordCount = values => {
```

Comment:
The function is never called anywhere 😅

Reply:
happy to remove it, I have no idea what it does!

Reply:
hey this is being called in `TimelineEditor`!

Reply:
It is the function that is responsible for displaying the message. You've actually removed it from the guide editor: was that intended? If yes sorry, I don't see what is picking up the word count then?

Reply:
ok, it's in the scribe editor now. Let's leave it there for now, but remember to clean it up when you're done moving the other two snippet types 🤗